device_calendar/lib/src/models/attendee.dart

```diff
@@ -43,7 +43,7 @@ class Attendee {
   }
 
   Map<String, dynamic> toJson() {
-    final Map<String, dynamic> data = Map<String, dynamic>();
+    final data = <String, dynamic>{};
```

Comment:
Merge this line with the next two to initialise the Map -  see https://www.tutorialspoint.com/dart_programming/dart_programming_map.htm